framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -5108,8 +5294,31 @@ VkResult VulkanReplayConsumerBase::OverrideAcquireNextImageKHR(PFN_vkAcquireNext
 
             assert(replay_index != nullptr);
 
-            result = swapchain_->AcquireNextImageKHR(
-                func, device_info, swapchain_info, timeout, semaphore_info, fence_info, captured_index, replay_index);
+            VkSwapchainKHR swapchain = VK_NULL_HANDLE;
+            VkSemaphore    semaphore = VK_NULL_HANDLE;
+            VkFence        fence     = VK_NULL_HANDLE;
+            if (swapchain_info != nullptr)
```

Comment:
In the begin of `OverrideAcquireNextImageKHR`, it has checked `swapchain_info != nullptr`. We don't need to check again.